homeassistant/components/tuya/switch.py

```diff
@@ -480,6 +480,46 @@
             name="Switch",
         ),
     ),
+    # Fan
+    # https://developer.tuya.com/en/docs/iot/categoryfs?id=Kaiuz1xweel1c
+    "fs": (
+        SwitchEntityDescription(
+            key=DPCode.ANION,
+            name="Anion",
+            icon="mdi:atom",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.HUMIDIFIER,
+            name="Humidifier",
+            icon="mdi:air-humidifier",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.OXYGEN,
+            name="Oxygen",
+            icon="mdi:molecule",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.FAN_COOL,
+            name="Fan Cool",
+            icon="mdi:sun-snowflake",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
What does this do?

Reply:
It's some fan function for cool wind (based on documentation only).

Reply:
I think we should leave this out for now. As it is not clear what it is; and therefore not sure how it should be represented or named.

Somehow this feels like some kind of air conditioning function intertwined with the fan platform :S

Reply:
Okay, I'll take it off.
I don't quite understand it either, it seems to be between a humidifier and an air conditioner.

Reply:
It's a pulsing mode, natural wind mode.